web/content/3.0.md

```diff
@@ -36,9 +36,51 @@ plugins. Official documentation [is available here](https://theforeman.org/manua
 
 ### Katello headline features
 
+For the full list of changes, see the [Changelog](https://github.com/Katello/katello/blob/KATELLO-4.2/CHANGELOG.md).
```

Comment:
I'd like to unify this process with Foreman's at some point for a more consistent user experience. However, I'm fine with doing that at a later point.